network/manager.go

```diff
@@ -382,6 +476,31 @@ func (nm *networkManager) GetEndpointInfo(networkId string, endpointId string) (
 	nm.Lock()
 	defer nm.Unlock()
 
+	if nm.IsStatelessCNIMode() {
+		cnilogger.Logger.Info("calling cns getEndpoint API")
+		endpointResponse, err := nm.CnsClient.GetEndpoint(context.TODO(), endpointId)
+		if err != nil {
+			cnilogger.Logger.Error("Get endpoint API returend with error", zap.Error(err))
+			return nil, errors.Wrapf(err, "Get endpoint API returend with error")
+		}
+		epInfo := &EndpointInfo{
+			Id:                 endpointId,
+			IfIndex:            0, // Azure CNI supports only one interface
```

Comment:
For ifindex, we should  return whatever existing cni returns.. if required we can get interface index based on interface name

Reply:
This value Has been hardcoded on existing GetEndpointInfo API with 0. I just followed the same. For the new commit, I just added a constant value.